pages/app/embed/EmbedWidget.js

```diff
@@ -2,36 +2,46 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import isEmpty from 'lodash/isEmpty';
 import d3 from 'd3';
-import { Autobind } from 'es-decorators';
-import { toastr } from 'react-redux-toastr';
-import { VegaChart } from 'widget-editor';
 
 // Redux
 import withRedux from 'next-redux-wrapper';
 import { initStore } from 'store';
 import { bindActionCreators } from 'redux';
 import { getWidget } from 'redactions/widget';
-import { setUser } from 'redactions/user';
-import { setRouter } from 'redactions/routes';
+import { setEmbed } from 'redactions/common';
 
 // Components
 import Page from 'components/app/layout/Page';
 import EmbedLayout from 'components/app/layout/EmbedLayout';
+import VegaChart from 'components/widgets/charts/VegaChart';
```

Comment:
We should use VegaChart from `widget-editor`. This is a change Clement inserted the other day.